daml/Tests/Distribution/Syndication/Util.daml

```diff
@@ -92,10 +92,10 @@ createBiddingService operator provider customer = do
   offerCid <- submit provider do exerciseByKeyCmd @Distributor.Role (operator, provider) Distributor.OfferBiddingService with ..
   submit customer do exerciseCmd offerCid Bidding.Accept
 
-createLifecycleService : Party -> Party -> Party -> Party -> Script (ContractId Lifecycle.Service)
-createLifecycleService operator issuer bondRegistrar cashProvider  = do
+createLifecycleService : Party -> Party -> Party -> Party -> Party -> Script (ContractId Lifecycle.Service)
+createLifecycleService operator issuer bondRegistrar cashProvider payingAgent = do
   offerCid <- submit issuer do exerciseByKeyCmd @Issuer.Role (operator, issuer) Issuer.OfferLifecycleService with ..
-  submit bondRegistrar do exerciseCmd offerCid Lifecycle.Accept
+  submitMulti [bondRegistrar, payingAgent] [] do exerciseCmd offerCid Lifecycle.Accept
```

Comment:
@dorrit-du-da since we are adding payingAgent into the Lifecycle Service Signatory, to create a LifeCycle we need consent from all operator issuer (obtained from the offer) and bondRegistrar, payingAgent (onbtained from accepting offer 
Does it look fine for you?

Reply:
@judywu-da, this is difficult to achieve on frontend level as you need two signatories to signoff which means you may induce extra contract/step in order to get both bond registrar and paying agent to sign off

Reply:
@dorrit-du-da I updated as discussed offline in commit 40c0f377facd3311f9c204d555dc860ef330c8e6